### PR TITLE
Add HI crypto icon

### PR DESCRIPTION
**Pull request by domhhv**
This PR adds a crypto icon for [$HI](https://hi.com/) currency support.

[Related issue](https://linear.app/unstoppable-domains/issue/ECH-717/new-currency-support-dollarhi)

Screenshot 2023-03-10 at 14 53 09

### CODE REVIEW

**Review by udhriaznov — APPROVED** [review]
LGTM

File: src/icons/crypto/HI.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    iconRef?: React.ForwardedRef<SVGSVGElement>;
+  } & SvgIconProps
+>(({iconRef, ...props}, ref) => {
+  return (
+    <SvgIcon viewBox="0 0 512 512" {...props} ref={iconRef || ref}>
+      <defs>
+        <clipPath id="clippath">
+          <circle fill="none" cx="256" cy="256" r="223.99" />
+        </clipPath>
+        <filter id="drop-shadow-1" filterUnits="userSpaceOnUse">
```
Comment (iurevych) [inline src/icons/crypto/HI.tsx:16]: Please consider the following:
1. Avoid raster graphics, it is better to take the SVG from their official website, and add some circle and change colors in Figma, to compose a simple icon which fits our standards (you can also ask Daniil)
2. Optimize the SVG code https://unstoppabledomains.slack.com/archives/C03702HJCN7/p1673957738038389

**Review by iurevych — CHANGES_REQUESTED** [review]

**Review by iurevych — APPROVED** [review]
👍

File: src/icons/crypto/HI.tsx (hunk 1 above)
Comment (iurevych) [reply src/icons/crypto/HI.tsx]: That can work

**Pull request merged**